torch/_dynamo/optimizations/torchxla_integration.py

```diff
@@ -137,12 +136,12 @@ def extract_compiled_graph(model: torch.fx.GraphModule, example_inputs):
 
```

Comment:
I prefer you rename `ivalue` to `xla_value` or `device_data` (something to avoid confusing people, since this is _not_ using torchscript

Reply:
I'll call it xla_value then.